src/lib/trivy/trivy-collector.spec.ts

```diff
@@ -7,12 +7,13 @@ describe('TrivyCollector', () => {
 
   let collector: TrivyCollector;
   let logger: Logger;
+  let theSpy;
```

Comment:
We shouldn't need to capture a reference to this. If we do want to, we should name it something less generic to prevent future naming collisions.